server.js

```diff
@@ -5,21 +5,18 @@ const app = require('./src/app');
 const env = nconf.get('env');
 
 // Check ES (if in use)
-if (env.ES_HOST && typeof env.ES_HOST === 'string') {
+if (env.ES_HOST) {
   const runES = require('./src/run-es');
   const checkES = new runES.Elastic();
-  checkES.setup()
-    .then(() => {
-      startListen();
-    });
+  checkES.setup();
 } else {
-  startListen();
-  logger.debug(`No (valid) Elasticsearch host provided - results will NOT be saved!`);
+  logger.debug(`[Elasticsearch] no host provided in config [${env.APP_CONFIG}] - results will NOT be saved!`);
```

Comment:
nit: these messages are inconsistent in their prefix spacing (`[Elasticsearch]` and `[ timings API ]`)